modules/atomic-widgets/elements/atomic-tabs/atomic-tabs.php

```diff
@@ -151,9 +154,29 @@ public function get_script_depends() {
 
 	protected function define_render_context(): array {
 		$default_active_tab = $this->get_atomic_setting( 'default-active-tab' );
+		
+		$direct_children = Collection::make( $this->get_children() );
+		
+		$tabs_menu = $direct_children->filter( fn( $child ) => $child->get_type() === 'e-tabs-menu' )->first();
+		$tabs_content_area = $direct_children->filter( fn( $child ) => $child->get_type() === 'e-tabs-content-area' )->first();
+		
+		$tabs_map = Collection::make( $tabs_menu ? $tabs_menu->get_children() : [] )
+			->filter( fn( $element ) => $element->get_type() === 'e-tab' )
+			->map( fn( $element ) => $element->get_id() )
+			->flip()
+			->all();
+		
+		$tabs_content_map = Collection::make( $tabs_content_area ? $tabs_content_area->get_children() : [] )
+			->filter( fn( $element ) => $element->get_type() === 'e-tab-content' )
+			->map( fn( $element ) => $element->get_id() )
+			->flip()
+			->all();
```

Comment:
Extract into function and pass the names of the elements as argument, and try use consts